pihole

```diff
@@ -497,8 +496,7 @@ Debugging Options:
 
 Options:
   setpassword           set the password for the web interface
-  -c, chronometer     Calculates stats and displays to an LCD
-                        Add '-h' for more info on chronometer usage
+  -c, chronometer     Chronometer is gone, use PADD
```

Comment:
I'm not sure if we should keep the `-c` option on the help function.

Reply:
You're probably right. They will get an info when actually requesting `-c` no need to keep it in help.